upstream-community-operators/crossplane/0.11.0/crossplane.v0.11.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,456 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ''
+          resources:
+          - events
+          verbs:
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ''
+          resources:
+          - secrets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+        - apiGroups:
+          - apiextensions.crossplane.io
+          - cache.crossplane.io
+          - compute.crossplane.io
+          - core.oam.dev
+          - database.crossplane.io
+          - kubernetes.crossplane.io
+          - packages.crossplane.io
+          - storage.crossplane.io
+          - workload.crossplane.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: crossplane
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: crossplane-package-manager
+      deployments:
+      - name: crossplane
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: crossplane
+          template:
+            metadata:
+              labels:
+                name: crossplane
+            spec:
+              containers:
+              - image: crossplane/crossplane:v0.11.0
+                imagePullPolicy: Always
+                name: crossplane
+              serviceAccountName: crossplane
+      - name: crossplane-package-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: crossplane-package-manager
+          template:
+            metadata:
+              labels:
+                name: crossplane-package-manager
+            spec:
+              containers:
+              - args:
+                - package
+                - manage
+                - --templates
+                - --templating-controller-image
+                - crossplane/templating-controller:v0.4.0
+                env:
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      apiVersion: v1
+                      fieldPath: metadata.namespace
+                image: crossplane/crossplane:v0.11.0
```

Comment:
In the version of this file submitted for 0.9.0, I had to add:
```yaml
-                - name: STACK_MANAGER_IMAGE
-                  value: crossplane/crossplane:v0.9.0
```

This was needer because the stack-manager could not detect the pod name when accompanied with a sidecar (which the OLM E2E testing environment added).  Since stack-manager could not detect the pod name, I had to provide it as an environment variable (a feature that I also used in debug/development when running the controller outside of Kubernetes (there is no pod)).

Reply:
This was only realized when testing that a stack install could succeed, in the E2E testing environment.

Reply:
This will be probably relevant in #1787  as well right?

Reply:
I would imagine so. Again, this only affected the E2E tests. It's possible the pod name detection in Crossplane has been dressed up since the last OperatorHub channel update to handle the existence of the sidecar.

Reply:
Thank you @displague, I really appreciate that extra context!